alacritty_terminal/src/term/mod.rs

```diff
@@ -1826,24 +1812,29 @@ impl<T: EventListener> Handler for Term<T> {
 
         if let Ok(bytes) = base64::decode(base64) {
             if let Ok(text) = str::from_utf8(&bytes) {
-                self.clipboard.store(clipboard_type, text);
+                self.event_proxy.send_event(Event::ClipboardStore(clipboard_type, text.to_owned()));
```

Comment:
If we use `String::from_utf8(bytes)` here, we can combine the two `if let` statements into a single one using `and_then`. Since we're doing `to_owned` anyways that should work out great.